bundle/manifests/lvm.topolvm.io_lvmclusters.yaml

```diff
@@ -301,10 +301,35 @@ spec:
                         properties:
                           devices:
                             description: Devices is the list of devices used by the
-                              deviceclass
+                              VG
                             items:
                               type: string
                             type: array
+                          excluded:
+                            description: Excluded contains the per node status of
+                              applied device exclusions that were picked up via selector,
+                              but were not used for other reasons. It can be consulted
+                              in case the device is not picked up as a device for
+                              LVM to use.
+                            items:
+                              properties:
+                                name:
+                                  description: Name is the device that was filtered
+                                  type: string
+                                reasons:
+                                  description: Reasons are the human-readable reasons
+                                    why the device was excluded and could not be used
```

Comment:
I would change it to "... why the device was excluded from the volume group"

Currently the sentence is saying the same thing twice (`excluded`, `could not be used`)